### PR TITLE
Chinese translation of documentations. [concepts and desing]

### PR DESCRIPTION
**Pull request by kezhenxu94**
To get the translation reviewed and improved early, I'll push here step by step, and merge finally after they are reviewed and improved enough.

Glad to receive your advices, any suggestions are welcome.

### CODE REVIEW

File: zh/concepts-and-designs/manual-sdk.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# 手动打点 SDK
```
Comment (wu-sheng) [inline zh/concepts-and-designs/manual-sdk.md]: 翻译不一致，上面写的手动埋点

File: zh/concepts-and-designs/service-agent.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# 服务自动打点代理
```
Comment (wu-sheng) [inline zh/concepts-and-designs/service-agent.md]: 同理，打点和埋点说法需要统一

**Comment by wu-sheng** [comment]
I think we could merge part of document, just not publish them in github page or community. Set them in several pr should be better.

**Comment by kezhenxu94** [comment]
> I think we could merge part of document, just not publish them in github page or community. Set them in several pr should be better.

Will separate them into groups of prs as per content, thanks

**Comment by wu-sheng** [comment]
Merge them when you think so. You don't need my permission. If you have questions about some translation, I could help at there.

**Comment by JaredTan95** [comment]
Hope those deleted files can help you. https://github.com/apache/incubator-skywalking/pull/2315/files?short_path=978e849#diff-978e84905a7d46e701e71ef5d9e57a5b

**Pull request merged**